src/commands/push-to-app-collection.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  app_name:
+    type: "string"
+  app_catalog:
+    type: "string"
+  app_collection_repo:
+    type: "string"
+steps:
+  - run: |
+        architect project version > version
+  - run: |
+        echo -n "<<parameters.app_name>>-$(cat version)" | head -c 53 > cr-name
+  - run: |
+        architect create appcr \
+            --name $(cat cr-name) \
+            --app-name <<parameters.app_name>> \
+            --app-version $(cat version) \
+            --catalog <<parameters.app_catalog>> \
+            -o yaml \
+          > app.yaml
```

Comment:
I'm uncertain how it will look in the CI but I would make it a single line just to make sure it will look good.

Reply:
It looks the same as here. see: https://circleci.com/gh/giantswarm/aws-operator/23508

Reply:
fixed